tests/integration/forgejo/test_issues.py

```diff
@@ -0,0 +1,82 @@
+# Copyright Contributors to the Packit project.
+# SPDX-License-Identifier: MIT
+from requre.online_replacing import record_requests_for_all_methods
+
+from ogr.abstract import IssueStatus
+from tests.integration.forgejo.base import ForgejoTests
+
+
+@record_requests_for_all_methods()
+class Issues(ForgejoTests):
```

Comment:
I guess this is mostly for @majamassarini as she started the parametrized tests for forgejo without any classes as we did before, I'm leaning towards keeping that convention rather than falling back to what we had before, but I also have to admit I don't remember how are the latest Forgejo tests done